packages/web/src/components/premium-content-purchase-modal/PremiumContentPurchaseModal.tsx

```diff
@@ -115,12 +112,16 @@ export const PremiumContentPurchaseModal = () => {
     }
   }, [stage, track, dispatch])
 
+  const mobile = isMobile()
+
   return (
-    <Modal
+    <ModalDrawer
       isOpen={isOpen}
       onClose={handleClose}
       onClosed={onClosed}
       bodyClassName={styles.modal}
+      isFullscreen
```

Comment:
Ditto on these props for mobile.

Reply:
since <ModalDrawer> handles the switching between web and mobile, these probs do need to get passed. Down to revisit this pattern afterwards though